assets/prototype/domain/shared/predicates/tickets/isExpired.ts

```diff
@@ -10,10 +10,14 @@ import { now } from './filters';
 import isValidEntityOrArchive from './isValidEntityOrArchive';
 import { Ticket } from '../../../eventEditor/data/types';
 
-const isExpired = (ticket: Ticket, includeArchived: boolean = false): boolean => {
+const isExpired = (ticket: Ticket, includeArchived = false): boolean => {
```

Comment:
Fix for:
![14_jan_3](https://user-images.githubusercontent.com/1477453/72342700-ca687a00-36d5-11ea-9d2c-26c28942fcbd.png)